src/DatePickerInput/web/DatePickerStyle.js

```diff
@@ -0,0 +1,76 @@
+import { createGlobalStyle } from 'styled-components';
+import reactDayPickerStyles from '!!raw-loader!react-day-picker/lib/style.css'; // eslint-disable-line
+
+const DatePickerStyle = createGlobalStyle`
```

Comment:
Let's call all the new global style components as Styles*
So this would be
`DatePickerStyles`

Reply:
I thought of doing that but doesn't `DatePickerStyles` sounds weird? Because it's a single component